docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -236,12 +241,54 @@ In words, our intervention algorithm does the following:
     approach the value :math:`c`, i.e. the proportion of people who eat the
     vehicle.
 
+Determining Whether a Simulant Is Affected
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Folic acid `reduces the birth prevalence <effect size_>`_ of :ref:`neural tube
+defects (NTDs) <2017_cause_neural_tube_defects>`. In order for a newborn to have
+a reduced risk of NTDs, the baby's mother needs sufficient folate intake *three
+months prior to pregnancy*. Therefore, only babies born to mothers who have been
+eating folic-acid-fortified food for at least a year will have any benefit from
+our intervention.
+
+With this in mind, here is the algorithm to determine whether a simulant is affected by the folic acid intervention:
+
+1.  **Define variables:** Each simulant needs a boolean attribute
+    `mother_ate_folate_fortified_food`, which will be `True` if the simulant's
+    mother ate folic-acid-fortified food starting at least a year before the
+    simulant was born, and `False` otherwise.
+
+2.  **Initialize the simulation:** At the start of the simulation, set
+    `mother_ate_folate_fortified_food = False` for all simulants; this attribute
+    will not be used after the simulant is born, so the value is irrelevant for
+    all simulants already alive at the start of the simulation.
```

Comment:
Ah, I think I realized why James wanted to get this part right... although this value is irrelevant to DALYs averted, setting it to False for all will result in an *over-estimate* of NTD DALYs in the baseline scenario.  The magnitude of this over-estimate will depend on the baseline coverage level, and we must remember that we have it when we do our standard validations or if we decide to calculate percent of DALYs averted.

Reply:
Setting it to false will not produce an over estimate.  The effect is on `birth_prevalence`, whereas sims initialized will use `prevalence`.  We should still be able to properly validate NTDs in the baseline and new cases coming in should properly be affected by the baseline coverage of folic acid fortification.  The limitation here is that we will simply not know the status of the mother for existing sims.  We can be explicit (and I will in the implementation) by saying `mother_ate_folate_fortified_food` is in the set {`True`, `False`, `Unknown`}.

Reply:
@collijk Sounds good. "Unknown" is more accurate than arbitrarily assigning True or False. Technical question: How would you implement this variable as a 3-value data type?  Do you just use strings?

@aflaxman Does James's explanation here sound accurate, i.e. that we won't overestimate DALYs in baseline?

It sounds like there are some issues here that should go in the limitations section.  Is there anything relevant here that should also be described in the algorithm?

Reply:
Yep, just use strings.